src/content/posts/2025-pyladiescon-is-back.mdx

```diff
@@ -0,0 +1,100 @@
+---
+title: "PyLadiesCon is back!"
+meta_title: "PyLadiesCon is back!"
+description: "PyLadiesCon returns on December 5–7 2025"
+date: 2025-06-23T23:00:00Z
+image: "/images/posts/pyladiescon-is-back.png"
+categories: ["Blog Post",]
+authors: ["Georgi"]
+tags: ["python", "community", "conference", "pyladies"]
+draft: false
+---
+
+
+## Save the Date 📅
+Yes! You read it right—**PyLadiesCon** returns December 5–7, 2025. 100% online conference designed for our global community. 
+Talks, workshops, panels, and community fun, all within the comfort of wherever your Wi‑Fi works. 
+
+---
+
+## Website is Live 🌐
+
+The official PyLadiesCon 2025 website is now live! 🎉
+
+Visit [our website](https://conference.pyladies.com/) to explore the new 2025 website, information behind the scene in our documentation or stay tuned for more updates, and announcements on our blog.
```

Comment:
Remove 
```suggestion

```

Reply:
Yeah I think we should still remove this paragraph, I felt it's confusing. I suggested the above list as a replacement (which has the same information mentioned by this paragraph, eg the documentation and the blog, and the 2025 website.

I would suggest adding the following paragraph:

```
The official PyLadiesCon website is now live! 🎉 
We've re-designed our conference website with many new information and documentation.
- For general information about our conference, visit our new landing page at https://conference.pyladies.com
- Visit the conference website for 2025 at .. (continue with the list above)
```

Reply:
updated